### PR TITLE
update hypershift-guest 4.12

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by heliubj18** [comment]
The cucushift is not ready now. just add it here, no case would be selected.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -392,12 +392,18 @@ tests:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and @hypershift-hosted'
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:395]: Also need `and not @fips`

**Comment by liangxia** [comment]
ci/prow/ci-operator-config-metadata — Job failed.

**Pull request merged**